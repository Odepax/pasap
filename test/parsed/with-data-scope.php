<?php use Pasap\Pasap; ?>

<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Pasap Test</title>
	</head>
	<datascope:parent <?= Pasap::scope([
		"number" => 12,
		"name" => "big bro",
		"wow" => "amaze"
	]) ?>>
		<datascope:child <?= Pasap::scope([
			"name" => "lit man",
			"more" => "fun"
		]) ?> />
	</datascope:parent>
	<datascope:child <?= Pasap::scope([
		"name" => "yay",
		"number" => 17
	]) ?> />
</html>
