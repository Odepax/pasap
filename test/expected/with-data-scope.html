<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Pasap Test</title>
	</head>
	<h1>Parent</h1>
	<p>Scope is:</p>
	<ul>
		<li><b>number:</b> 12</li>
		<li><b>name:</b> big bro</li>
		<li><b>wow:</b> amaze</li>
		<li><b>more:</b> </li>
	</ul>
	<h1>Child</h1>
	<p>Scope is:</p>
	<ul>
		<li><b>number:</b> 12</li>
		<li><b>name:</b> lit man</li>
		<li><b>wow:</b> amaze</li>
		<li><b>more:</b> fun</li>
	</ul>
	<h1>Child</h1>
	<p>Scope is:</p>
	<ul>
		<li><b>number:</b> 17</li>
		<li><b>name:</b> yay</li>
		<li><b>wow:</b> </li>
		<li><b>more:</b> </li>
	</ul>
</html>
